Laud_Misc_610/CGH/_1.trig

```diff
@@ -28,8 +28,12 @@
     owl:sameAs <http://example.com/LL/lagin.trig#MiledEspane>;
     owl:sameAs <http://example.com/Rawl_B502/fursunnud_naile.trig#Mílid>.
 
+<< <#Militis>
+        rdfs:comment "This text is in Latin.  The predicate used ties the name to Old Irish specifically.  Maybe we should remove the specific tie to Old Irish and let it be any nominative" >>
+        prov:wasAttributedTo <https://orcid.org/0000-0002-7241-3264> .
```

Comment:
We could, although could it be useful to tie it to Old Irish for when IrishGen gets included in federated queries involving a variety of resources? If we add `@la` to the name, will that override the predicate (long shot)?